HW02_A_Graph-Fails---Viola-fix.md

```diff
@@ -0,0 +1,129 @@
+What went wrong?
```

Comment:
I can see the graphs now! I have to go to the top right corner of this file with the "...", click the dots and then say "see file" and it brings me to a github page with the graphs on them. Nice job!

Reply:
Cool! Thank you for pointing it out. It makes it much easier to review!